package.json

```diff
@@ -27,10 +27,10 @@
   "homepage": "https://github.com/Microsoft/typed-rest-client#readme",
   "devDependencies": {
     "@types/mocha": "^2.2.44",
-    "@types/node": "^8.0.19",
+    "@types/node": "^6.12.0",
     "@types/shelljs": "0.7.4",
     "mocha": "^3.5.3",
-    "node": "^8.0.19",
```

Comment:
I understand the @types/node above, but what purpose does this line address?

Reply:
I thought we could use it to require a certain version of node for development? I know there is an "engines" option but I believe that is for execution and not development (though we may want to add that too). Is it sufficient to only reference node using the @ type?

Reply:
Yeah, I don't think it works that way.  If you want to do that, you should do it in the make.js file.  Let's remove that line and then do a separate PR for that.

Actually, I'm going to merge this and then do a checkin that removes it.